graylog2-web-interface/src/components/event-notifications/event-notification-types/EmailNotificationForm.jsx

```diff
@@ -179,6 +183,14 @@ class EmailNotificationForm extends React.Component {
             {lodash.get(validation, 'errors.recipients[0]', 'Add email addresses that will receive this Notification.')}
           </HelpBlock>
         </FormGroup>
+        <Input id="notification-time-zone"
+               help="Time zone used for timestamps in the email body."
+               label={<ControlLabel>Time zone for date/time values <small className="text-muted">(Optional)</small></ControlLabel>} >
```

Comment:
Another issue here: Adding `ControlLabel` results in having two `label` elements in the HTML. The easiest fix is to use a react fragment instead:

```suggestion
               label={<>Time zone for date/time values <small className="text-muted">(Optional)</small></>}>
```